acme/acme/challenges.py

```diff
@@ -54,43 +50,45 @@ class SimpleHTTP(DVChallenge):
 
     """
     typ = "simpleHttp"
-    token = jose.Field("token")
+
+    TOKEN_SIZE = 128 / 8  # Based on the entropy value from the spec
+    """Minimum size of the :attr:`token` in bytes."""
+
+    # TODO: acme-spec doesn't specify token as base64-encoded value
+    token = jose.Field(
+        "token", encoder=jose.encode_b64jose, decoder=functools.partial(
+            jose.decode_b64jose, size=TOKEN_SIZE, minimum=True))
+
+    @property
+    def good_token(self):  # XXX: @token.decoder
+        """Is `token` good?
+
+        .. todo:: acme-spec wants "It MUST NOT contain any non-ASCII
```

Comment:
If this method is going to validate that the token not contain non-ASCII characters, it should do so now, or else add a TODO in caps to a comment so we can find it easily by searching.